src/js/utils/browser.js

```diff
@@ -32,8 +32,13 @@ export function isMSIE() {
     return userAgentMatch(/msie/i);
 }
 
+export function isTizen() {
+    return userAgentMatch(/SMART-TV/);
```

Comment:
this is different than the approach above where we look for `Tizen`, can i get a quick explainer?

Reply:
"Tizen" is not present in the pre-2017 Smart TV user-agents. `SMART-TV` indicated it is Tizen. The word _Tizen_ is found before the Tizen _version_ in later models.